rbx_xml/test-files/inf-and-nan.rbxmx

```diff
@@ -7,25 +7,125 @@
 			<string name="Name">Folder</string>
 			<BinaryString name="Tags"></BinaryString>
 		</Properties>
-		<Item class="NumberValue" referent="RBX3B3D9D3DB43D4E6793B190B081E0A886">
+		<Item class="Folder" referent="RBX84B85A2C500048E79846CD63A6127BA3">
```

Comment:
Can we avoid introducing a grouping for just the NumberValue instances and keep everything flat?